### PR TITLE
Refactor identify kml

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

Follows feedback from release testing, refer to issue 7521.

- Not a perfect fix but adds some sizing to the VBox and Callout to improve the display of the text being read from the HTML.
- Updates the image to show all the text too (plus scrollbar)

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by jenmerritt** [comment]
@Rachael-E could I get a first review please?

**Review by Rachael-E — APPROVED** [review]
looks good to me @jenmerritt thanks.

**Comment by jenmerritt** [comment]
@Luiy0 second review please thanks!

File: kml/identify-kml-features/src/main/java/com/esri/samples/identify_kml_features/IdentifyKMLFeaturesSample.java [hunk 1]
```diff
@@ -41,6 +31,15 @@
 import com.esri.arcgisruntime.mapping.view.MapView;
 import com.esri.arcgisruntime.ogc.kml.KmlDataset;
 import com.esri.arcgisruntime.ogc.kml.KmlPlacemark;
+import javafx.application.Application;
```
Comment (Luiy0) [inline kml/identify-kml-features/src/main/java/com/esri/samples/identify_kml_features/IdentifyKMLFeaturesSample.java:34]: Could possibly add a newline here separating the imports.

**Review by Luiy0 — APPROVED** [review]
LGTM @jenmerritt

**Pull request merged**